setup.py

```diff
@@ -1,5 +1,7 @@
 from setuptools import setup, find_packages
-from text2term.config import VERSION
+import re
+
+VERSION = '4.3.1'
```

Comment:
I'm wondering what is the rationale behind this change. You see, t2t.py (in [L359](https://github.com/synthesizebio/text2term/blob/6a6b8b7044131401577038e0dfb18f13476a0deb/text2term/t2t.py#L359)) needs to read the VERSION value.  But maybe there is way to read it from `setup.py` that I'm not aware of? What do you think?